docs/docs-content/integrations/vault.md

```diff
@@ -12,94 +12,115 @@ tags: ['packs', 'vault', 'security']
 
 [Vault](https://www.vaultproject.io/) helps secure, store, and tightly control access to tokens, passwords, certificates, encryption keys for protecting secrets, and other sensitive data using a UI, CLI, or HTTP API.
 
+Vault integration has the following components:
+
+* Vault server
+* UI (optional).
+* [Agent injector](https://www.vaultproject.io/docs/platform/k8s/injector/) (optional).
+
 ## Versions Supported
 
 <Tabs queryString="versions">
 
 <TabItem label="0.22.x" value="0.22.x">
 
-* **0.22.0**
+### Prerequisites
 
-</TabItem>
+- A Kubernetes cluster with Kubernetes version 1.22 or later.
 
-<TabItem label="0.20.x" value="0.20.x">
+### Parameters
 
-* **0.20.1**
+The table lists commonly used parameters you can configure when adding this pack.
 
-</TabItem>
-<TabItem label="0.11.x" value="0.11.x">
+| Parameters | Description | Default |
+-------------|-------------|---------|
+|`charts.server.ingress` | Enable ingress traffic to the Vault server. If you want to enable ingress traffic, make sure that `charts.server.serviceType` is set to `"ClusterIP"` or is left empty. | `false` |
+|`charts.global.tlsDisable` | Disable TLS for end-to-end encrypted transport. | `true` |
+|`charts.agent.enabled` | Enable vault agent injection to inject secrets into the pods. | `-` |
+|`charts.server.ha` | Enable high-availability mode to protect against outages by running multiple Vault servers. For more information, refer to [Vault documentation](https://developer.hashicorp.com/vault/docs/internals/high-availability). |  `false` |
+|`charts.server.dataStorage`| Controls the size, location, storage class of the persistent storage used by the Vault. | |
 
-* **0.11.0**
+:::caution
```

Comment:
This caution is very specific to RKE2. Could we create a section in usage for maybe Storage and mention this under that section? I get the sense we might have more of this challenges related to storage in the future.